CMakeLists.txt

```diff
@@ -533,6 +533,18 @@ if(H3_IS_ROOT_PROJECT AND BUILD_TESTING)
         endif()
     endmacro()
 
+    macro(add_h3_cli_test name h3_args expect_string)
+        add_test(NAME ${name}_test${test_number}
+                 # /usr/bin/sh -c "test \"`./bin/h3 cellToLatLng -c 8928342e20fffff`\" = \"37.5012466151, -122.5003039349\""
+                 COMMAND ${SHELL} "test \"`./bin/h3 ${h3_args}`\" = '${expect_string}'")
```

Comment:
We may need to check for `test` being available and error the CMake configuration if not.

Reply:
Discussed offline. `test` is defined because we depend on git for windows, which brings in bash, but `./bin/h3` doesn't work because it's `./bin/h3.exe` there.